web/src/js/ducks/ui.js

```diff
@@ -1,5 +1,5 @@
-import { SELECT_FLOW } from './flows'
-const SET_ACTIVE_MENU = 'SET_ACTIVE_MENU'
+export const SELECT_FLOW = "SELECT_FLOW"
```

Comment:
i changed it in the new commit, but i got an error with i cannot solve:
`Error: Cannot find module '../components/FlowTable/FlowColumns' from 'flows.js'`

if i change this line in js/ducks/flows.js 
`import * as columns from "../components/FlowTable/FlowColumns";
`to
`import * as columns from "../components/FlowTable/FlowColumns.jsx";
`
i got this error:
```
  - TypeError: Cannot read property 'reduceList' of undefined
        at Object.<anonymous> (src/js/ducks/flows.js:103:1542)
```